doc/source/user_guide/geometrical_transform.rst

```diff
@@ -0,0 +1,132 @@
+============================================
+Geometrical transformations of images
+============================================
+
+Cropping, resizing and rescaling images
+---------------------------------------
+
+.. currentmodule:: skimage.transform
+
+Images being NumPy arrays (as described in the :ref:`numpy` section), cropping
+an image can be done with simple slicing operations. Below we crop a 100x100
+square corresponding to the top left corner of the astronaut image. Note that
+this operation is done for all color channels (the color dimension is the last
+third dimension):: 
+
+   >>> from skimage import data
+   >>> img = data.astronaut()
+   >>> top_left = img[:100, :100]
+
+In order to change the shape of the image, :mod:`skimage.color` provides several
+functions described in :ref:`sphx_glr_auto_examples_transform_plot_rescale.py`
+.
+
+.. literalinclude:: ../../examples/transform/plot_rescale.py
+    :language: python
+    :start-after: import matplotlib.pyplot as plt
+    :end-before: fig, axes
+
+
+.. image:: ../auto_examples/transform/images/sphx_glr_plot_rescale_001.png
+   :target: ../auto_examples/transform/plot_rescale.html
+   :align: center
+   :width: 80%
```

Comment:
I'm lost over here. The image I see is a 4x4 grayscale of the full version of the astronaut. It's not cropped, and not in color. Is this intended?

Reply:
This example shows how to use the functions `rescale`, `resize` and `downscale_local_mean`. It is not for cropping but rather changing the total size of the image. Do you think I should change the wording "n order to change the shape of the image"?